### PR TITLE
v0.14

### PR DESCRIPTION
**Pull request by kejace**
Generated files have been tested with `chanterelle` and `chanterelle-halogen-template`.

### CODE REVIEW

**Comment by kejace** [comment]
I have tested this with `chanterelle-halogen-template` and it works in the browser.

File: src/Data/Generator.purs [hunk 1]
```diff
@@ -349,7 +349,7 @@ toTransportPrefix isConstructor isCall outputCount = do
   pure $ modifier <> fun
 
 toPayload :: Boolean -> String -> Array String -> Imported String
-toPayload isWhereClause typeName args = do
```
Comment (blinky3713) [inline src/Data/Generator.purs:352]: if toPayload can be redefined without taking this argument, we should do that

File: src/Data/Generator.purs [hunk 2]
```diff
@@ -303,7 +303,7 @@ funToHelperFunction' fun@(SolidityFunction f) opts = do
         ty <- toPSType fi.type
         pure $ fi.name <> " :: " <> ty
       pure $ "{ " <> joinWith ", " rowElems <> " }"
-    whereHelper d pre is ret = do
+    whereHelper _ pre _ ret = do
```
Comment (blinky3713) [inline src/Data/Generator.purs:306]: are these arguments being used anywhere? If not we should remove them entirely

File: src/Data/Generator.purs [hunk 3]
```diff
@@ -146,7 +146,7 @@ data FunTypeDecl =
               }
 
 funToTypeDecl :: SolidityFunction -> CodeOptions -> Imported FunTypeDecl
-funToTypeDecl fun@(SolidityFunction f) opts = do
+funToTypeDecl fun@(SolidityFunction f) _ = do
```
Comment (blinky3713) [inline src/Data/Generator.purs:149]: Can we remove the `CodeOptions` arg and eveyrthing still work?

**Review by blinky3713 — CHANGES_REQUESTED** [review]

**Review by blinky3713 — APPROVED** [review]

**Pull request merged**